docs/getting-started.md

```diff
@@ -285,11 +287,11 @@ To build for the Jetson Nano, build the target with the `--cpu=jetson_nano` flag
 
 ## Deploying to Jetson Nano 
 
-We use ansible to automatically update software running on the Jetson Nano. [See these instructions.](/docs/deploying.md) 
+We use ansible to automatically update software running on the Jetson Nano. [More info here.](useful-robot-commands.md#flashing-the-nano) 
 
 To update binaries on a working robot, you can run:
 
-`./tbots.py run run_ansible --cpu=jetson_nano -- --playbook remote_flash.yml --port 45000 --ssh_pass our_password_here`
+`./tbots.py run run_ansible --cpu=jetson_nano -- -playbook deploy_nano.yml --hosts <robot_ip> --ssh_pass thunderbots`
```

Comment:
I think we should bring back `our_password_here` (like the command in [robot-software-architecture.md](https://github.com/UBC-Thunderbots/Software/pull/2762/files#diff-f04984ac1e564b9b42774779bdfc5edfe47c3edc7e2f85fe5cfa4a86130a1437))